chart/compass/charts/director/config.yaml

```diff
@@ -19,9 +19,13 @@ graphql:
     integrationSystems: ["integration_system:read"]
     viewer: []
     tenants: ["tenant:read"]
+    tenantByInternalID: ["ory_internal"]
```

Comment:
My thought was the same. But I saw we have other internal scopes with this read/write separation - application, runtime, app_template, etc. Also, in the GraphQL scopes file it will be consistent with the other entries where all queries have `..:read` scope and all of the mutation `..:write` scope. FMPOV it doesn't matter, if we made that separation it will be only for consistency, from functional/technical aspect there is no difference